pkg/util/exec/exec.go

```diff
@@ -199,6 +199,9 @@ func (*CommandExecutor) ExecuteCommandWithOutputFile(command, outfileArg string,
 	// #nosec G204 Rook controls the input to the exec arguments
 	cmd := exec.Command(command, arg...)
 	cmdOut, err := cmd.CombinedOutput()
+	if err != nil {
+		cmdOut = []byte(fmt.Sprintf("%s. %s", string(cmdOut), string(err.(*exec.ExitError).Stderr)))
```

Comment:
Thought: Should `stderr:` be added as a prefix to the stderr output?

Reply:
What about just logging it directly instead of appending to cmdOut? 
```suggestion
		logger.Debug("stderr: %s", string(err.(*exec.ExitError).Stderr)))
```

Reply:
In what scenario did you find this useful? Perhaps in the osd prepare job where debug logging is enabled by default.

Reply:
It is useful in all commands using `ExecuteCommandWithOutputFile` so 90% of them. If we don't do this the error only prints like so:

```
 failed to check if we can stop the deployment rook-ceph-osd-2: failed to check if rook-ceph-osd-2 was ok to stop: deployment rook-ceph-osd-2 cannot be stopped: exit status 16
```

And the real error is always hidden unless we turn on debug logging.

Reply:
Sounds good, I see now we're returning it as part of cmdOut instead of just printing with debug logging.

Reply:
Do we always get an `ExitError`? We may need to check to make sure it's that type.